### PR TITLE
New hierarchy changes

### PR DESCRIPTION
**Pull request by senseibara**
- [x] User configuration 
- [x] Moving contact to appropriate position 
- [x] Update forms with assignment

### CODE REVIEW

File: niger/tasks.js [hunk 1]
```diff
@@ -11,6 +11,32 @@ const descriptiveContactLabel = (contact, report) => {
   return label;
 };
 
+const completeEditableContent = (content, contact, report) => {
+  content.g_details = {};
+  content.g_symptoms = {};
+  content.g_details.case_number = Utils.getField(report, 'g_alterations.alt_case_number') || Utils.getField(report, 'g_details.case_number');
+  content.g_details.lastname_of_patient = Utils.getField(report, 'g_alterations.alt_lastname_of_patient') || Utils.getField(report, 'g_details.lastname_of_patient');
+  content.g_details.firstname_of_patient = Utils.getField(report, 'g_alterations.alt_firstname_of_patient') || Utils.getField(report, 'g_details.firstname_of_patient');
+  content.g_details.age = Utils.getField(report, 'g_alterations.alt_age') || Utils.getField(report, 'g_details.age');
```
Comment (kennsippell) [inline niger/tasks.js]: This code seems stringy and has a larger than necessary surface area for bugs...

Can we handle this generically? Maybe something like:

```
const attributes = {
  g_details: ['lastname_of_patient', 'firstname_of_patient', ...],
  g_symptoms: ['symptoms'],
};
const sources = [report.fields.g_alterations, report.fields.g_details, ...];
const optionalPrefixes = ['g_', 'alt_'];

for each attribute
  const availablePaths = sources * prefixes;
  content[prefix][attribute] = availablePaths.find(path => Utils.getField(path));
```

File: niger/forms/app/samu.xml [hunk 2]
```diff
@@ -1021,7 +1021,7 @@
           <send_to_riposte tag="hidden"/>
           <g_details tag="hidden">
             <covid_call tag="hidden"/>
-            <riposte_id tag="hidden">a19011ff-d673-403a-bb33-46891a61493e</riposte_id>
+            <riposte_id tag="hidden">7ee82239-5d4d-4181-959c-584ab1658358</riposte_id>
```
[the remark this answers is not in the document]
Comment (senseibara) [reply niger/forms/app/samu.xml:1024]: Sorry about this i am just changing it to upload it to the test instance then changing it back to normal. the XLSX have the correct id but not the XML that haven't been recompiled yet.

File: niger/tasks.js [hunk 3]
```diff
@@ -11,6 +11,32 @@ const descriptiveContactLabel = (contact, report) => {
   return label;
 };
 
+const completeEditableContent = (content, contact, report) => {
+  content.g_details = {};
+  content.g_symptoms = {};
+  content.g_details.case_number = Utils.getField(report, 'g_alterations.alt_case_number') || Utils.getField(report, 'g_details.case_number');
```
[the remark this answers is not in the document]
Comment (senseibara) [reply niger/tasks.js]: group

File: niger/tasks.js (hunk 1 above)
Comment (senseibara) [reply niger/tasks.js]: Thanks for this snippet. Will apply

Comment (senseibara) [reply niger/tasks.js]: putting a code improvement/cleaning note to the board to do this in a second time and focus on the P0 for this week.

**Pull request merged**